### ISSUE TITLE
Feature request: Seasonal color options

### ISSUE THREAD

**Issue by tali713** [Amostubal/modified-MWDF#52]
Though I usually just play with seasonal colors disabled…  In addition to having a toggle for seasonal colors it might be nice to have drop down to select variant seasonal colors.  Ideally with them as the default for their related color theme, though this is hardly necessary.

See the [Seasonal Colors thread]( http://www.bay12forums.com/smf/index.php?topic=161494.0) on bay12 forums.

**Comment by Amostubal** [comment]
@tali713 been a quiet few days from you.  hmmm... well I was actually was happy about getting the switch in, thanks for noticing... let me see how it switches the colors currently... then people can submit their seasonal set with their color sets... we could mark the color name with a * in the color selector and add to the tool tip that * means it has an included "seasonal pallete" and we just leave the last pallete in if they switch to one that does not have a seasonal pallete.

**Comment by tali713** [comment]
And yeah, noticed right away, I've been manually disabling it for a while now, so the change was greatly appreciated.

**Comment by Amostubal** [comment]
great! I'm exhausted... I hate to say there is about to be several files changed in the morning... I'm too tired to hit the button right now... I might send in something that isn't right.

**Comment by Amostubal** [comment]
Well I've looked at it a few times.  and I'm not sure how to incorporate it yet... I plan to open a repository of the VB script at the 1.28 release, along with a complete rework of this github to be more professional looking now that I understand github better(no separate branches, etc).  I really hate VB... I did some work in it literally DECADES ago and honestly its no better now then it was then.

**Comment by tali713** [comment]
Yeah, git/github takes a minute to get used to and comfortable with, even if you cheat like I do and use the [magit](https://magit.vc) package for emacs.  As for VB, totally, but if you get the source you are currently using up, I would be happy to see what I can do.

**Comment by Amostubal** [comment]
New format of the github, just let you know, tali713.  After I pack this release. and upload it to DFFD I'm going to begin uploading the mass that is the VB folder.
